lab_3_ann_retriever/main.py

```diff
@@ -149,6 +185,11 @@ def __init__(self, corpus: list[list[str]]) -> None:
         Args:
             corpus (list[list[str]]): Tokenized documents to vectorize
         """
+        self._corpus = corpus
+        self._vocabulary = []
+        self._idf_values = {}
+        self._token2ind = {}
+        self.build()
```

Comment:
do yoy really build here?

Reply:
without it there is an error in test_vectorizer_return_value

Reply:
strange. You must not run build in the init

Reply:
agree